server/models/components/box_convolution.py

```diff
@@ -1,7 +1,46 @@
 import math
 from collections import deque
 from models.components.box import Box
-from models.components.utils import compute_remove_delta
+
+
+def remove_values_from_array(array, remove, integer_values):
+    """
+    given an array of values, remove values according to values_to_remove
```

Comment:
according to 'remove' plutôt, si on s'en tient aux noms des arguments de la fonction